docs/MAINTAINERS.md

```diff
@@ -1,44 +0,0 @@
-# For maintainers only
```

Comment:
I assume all of these deletes were decided elsewhere.

Reply:
yes, those docs were months/years out of date and the mc documentation exists on the website now.